development/cpp/binding_to_external_libraries.rst

```diff
@@ -75,7 +75,12 @@ need to be created:
     register_types.h
     register_types.cpp
 
-With the following contents:
+.. note::
+    **Important:** These files must be in the top level folder of your module
```

Comment:
There's a Sphinx admonition for that™ :wink: 

```suggestion
.. important::
    These files must be in the top-level folder of your module
```

Reply:
Aha! I'm new to Sphinx (Godot seems to use a number of tools I've never used before :-))

I was following what was used below - do you want me to change those as well?

Reply:
There's also a `**note**:` in `custom_modules_in_cpp.rst` - change that one to `.. note::`?

Reply:
> I was following what was used below - do you want me to change those as well?

Yes, it'd be better to change them as well for consistency.

> There's also a **note**: in custom_modules_in_cpp.rst - change that one to .. note::?

Sure :slightly_smiling_face: